CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ Please remember our [Code of Conduct](https://exercism.io/code-of-conduct) when
 These are our style guidelines which are constantly evolving.
 
 ### Ruby Code
-- Use brackets in Ruby. e.g. `some_method(arg1, arg2)` rather than `some_method arg1, arg2`
+- Use parenthesis in Ruby. e.g. `some_method(arg1, arg2)` rather than `some_method arg1, arg2`
```

Comment:
I think the right spelling might actually be `parentheses` (see [this explanation](https://english.stackexchange.com/questions/130219/parentheses-vs-parenthesis#:~:text=Parenthesis%20is%20of%20course%20Greek,%22%20means%20%3A%20as%20an%20afterthought.)). But correct me if I'm wrong!

```suggestion
- Use parentheses in Ruby. e.g. `some_method(arg1, arg2)` rather than `some_method arg1, arg2`
```